app/components/point.jsx

```diff
@@ -4,6 +4,7 @@
 import React, { useState } from 'react'
 import cx from 'classnames'
 import { createUseStyles } from 'react-jss'
+import '../../assets/css/white_boxes.css'
```

Comment:
The CSS styles need to be incorporated into this file - in the createUseStyles object at the bottom.